docs/content/preview/migrate/known-issues/oracle.md

```diff
@@ -243,3 +244,33 @@ CREATE TABLE employees (
     PRIMARY KEY (employee_id, hire_date)
 ) PARTITION BY RANGE (hire_date) ;
 ```
+
+---
+
+### Negative scale is not supported
+
+**GitHub**: [Issue #779](https://github.com/yugabyte/yb-voyager/issues/779)
+
+**Description**: Oracle supports negative scale where you can round up the values to the power of tens corresponding to the scale provided. Negative scale support is not supported in PostgreSQL and therefore in YugabyteDB.
```

Comment:
1. The values are round down. 
2. Can we please reduce the redundancy by removing support from a place.
`Negative scale support is not supported`
Thank You